content/en/users/cloud-compute/openstack/_index.md

```diff
@@ -215,107 +215,64 @@ openstack server create --flavor <flavor> \
 
 ## Terraform
 
-[Terraform](https://terraform.io) supports EGI Cloud OpenStack providers by
-using valid access tokens for Keystone. For using this, just configure your
-provider as usual in Terraform, but do not include user/password information:
+[Terraform](https://registry.terraform.io/providers/terraform-provider-openstack/openstack/)
```

Comment:
I would keep the link to terraform, not the provider. Although the link to the provider is also useful and should be added